src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -368,6 +368,8 @@ static ucs_status_t uct_rdamcm_cm_ep_server_init(uct_rdmacm_cm_ep_t *cep,
 err_reject:
     uct_rdmacm_cm_reject(cep->id);
 err:
+    uct_rdmacm_cm_destroy_id(cep->id);
+    cep->id = NULL;
```

Comment:
do we need to set to null? we never do after calling uct_rdmacm_cm_destroy_id...

Reply:
technically no but since this is separated from `ctor` function, IMO better to do this to avoid uninitialized field in `cep` after the call.